### ISSUE TITLE
Linter Rule: `html-aria-role-must-be-valid` missing valid roles

### ISSUE THREAD

**Issue by SleeplessByte** [marcoroth/herb#1726]
I think the mdn list for roles or something like that was used, but this means that valid roles such as `graphics-symbol` are marked as incorrect.

See: https://www.w3.org/TR/graphics-aria-1.0/#roles

There could be more valid specs that are missed.

**Comment by marcoroth** [comment]
Hey @SleeplessByte, thanks for reporting this!

We used the [erblint-github](https://github.com/github/erblint-github) rule as the foundation for this. I'd assume that at the time this rule was originally written that the state of the rules could have been different from now.

If they are valid according to the spec today, we should definitely update the rule and add them.
